Landing page/README.md

```diff
@@ -0,0 +1,2 @@
+# Landing-page
+Microverse project
```

Comment:
please add a description to ReadMe document. A detailed readMe explains purpose of your project. Its the first page a developer and other users access first and let's you stand out.